specification/dimensions/availabilityzone.md

```diff
@@ -0,0 +1,30 @@
+# Availability Zone
+
+Availability Zones are physically separated and isolated areas within a region which provide high availability and fault tolerance.
+
+The AvailabilityZone column MUST be present in the billing data. This column MUST be of type String and MAY contain null values.
+
+## Column ID
+
+AvailabilityZone
+
+## Display name
+
+Availability Zone
+
+## Description
+
+Physically separated and isolated areas within a region which provide high availability and fault tolerance.
+
+## Content constraints
+
+| Constraint      | Value           |
+|-----------------|-----------------|
+| Column required | True            |
```

Comment:
If we make it optional...
```suggestion
| Column required | False           |
```

Reply:
Thanks Michael, just updated file after Squad 2 meeting